spring-data-multi-storage-transaction-sample/README.md

```diff
@@ -1,6 +1,6 @@
-# Spring Data Integration Sample with Multi-storage Transactions
+# Sample application of Spring Data JDBC for ScalarDB with Multi-storage Transactions
```

Comment:
I thought "Spring Data JDBC sample for ScalarDB" or "Spring Data JDBC for ScalarDB sample" was a bit confusing. So I started with "Sample application of" to make the title clearer.